readme.md

```diff
@@ -10,19 +10,23 @@ CMP requests, sends them to the server and checks if responses match expectation
 which includes all the transmitted and received messages in base64 and in human-readable form.
 
 Test cases are written in a domain-specific language (DSL) tailored for PKI and X509 certificate parsing. The provided
-scenarios evaluate *server* implementations of CMP and the CMP lightweight profile. However, the DSL can also be used
-for writing client-oriented tests.
+scenarios evaluate *server* implementations of CMP [RFC](https://datatracker.ietf.org/doc/draft-ietf-lamps-rfc4210bis/) and the CMP lightweight profile [RFC 9483](https://datatracker.ietf.org/doc/html/rfc9483). 
 
 Several usage scenarios are possible:
 - Run it "as is" and check whether your CMP server conforms to the specification.
 - Adapt the test suite to your needs, by writing test scenarios using the provided DSL.
+
+Additionally, the DSL can also be used for writing client-oriented tests. A usage scenario could be: 
 - Extend the DSL, adding new keywords and correcting errors in existing ones.
 
 What makes this test suite unique is the high-level notation it is written in, making the reports and test scenarios
 readable not only to software engineers, but also to PKI experts without programming experience. Another key benefit is
 the emphasis on replicability - a test report is sufficient for someone to understand exactly what was transmitted to
 the server and how the responses were processed.
 
+These instructions assume a Linux-based system, but it is designed to be used with other platforms as well. Occasionally, additional Windows commands are provided. 

```

Comment:
It is better to say "Debian-based Linux system", because the commands invoke `apt`, which is a Debian tool and is not available in other distros. In that case the commands would have to be adjusted to the distro at hand.